### PR TITLE
chore: update nuxt-module-builder to 0.8.0

### PR DESCRIPTION
**Pull request by huang-julien**
### 🔗 Linked issue

#92

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

package update with a build fix

### CODE REVIEW

File: src/runtime/registry/vimeo-player.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { watch } from 'vue'
-import type Vimeo from 'vimeo__player'
+import type Vimeo from '@vimeo/player'
```
Comment (harlan-zw) [inline src/runtime/registry/vimeo-player.ts:2]: cursed vimeo :laughing:

**Comment by harlan-zw** [comment]
This won't fix #92 akaik? The module options will still be broken for the `registry` key.

File: src/runtime/registry/vimeo-player.ts (hunk 1 above)
Comment (huang-julien) [reply src/runtime/registry/vimeo-player.ts:2]: Cursed indeed 😩

**Comment by huang-julien** [comment]
Yeah, will fix it today :)

**Pull request merged**